tests/unit/test_matrix.py

```diff
@@ -242,3 +243,16 @@ def it_knows_how_to_assemble_its_blocks(self, request):
             [bs._base_values, bs._subtotal_columns],
             [bs._subtotal_rows, bs._intersections],
         ]
+
+
+class Describe_SumSubtotals(object):
+    @pytest.mark.parametrize(("prop_name",), (("foo",), ("bar",)))
```

Comment:
These propnames would be measure names, so "weighted_counts" and "unweighted_counts" might be better choices than "foo" and "bar".